inkcut/job/view.enaml

```diff
@@ -295,21 +353,21 @@ enamldef MaterialDockItem(DockItem):
                     image = load_image('shape_square_height')
                 DoubleSpinBox:
                     value << to_unit(job.material.size[1], plugin.units)
-                    value :: job.material.size[1] = from_unit(change['value'],plugin.units)
+                    value :: try_update_setting(plugin, self, job.material.size, 1, change)
                     suffix << " "+plugin.units
                     maximum = 99999.9
                     #maximum << to_unit(job.material.size[1], plugin.units)
                     #single_step = 0.1
-                Label:
-                    text = ""
-                PushButton:
-                    text = "Read from device"
-                    tool_tip = "Get the material size by querying the device."
-                    clicked ::
-                        try:
-                            job.material.size = plugin.device.querySize()
-                        except NotImplementedError:
-                            pass
+#                Label:
```

Comment:
Maybe create a github issue to re-introduce it and link to it from here?

Reply:
Good idea